src/storage/src/hummock/event_handler/hummock_event_handler.rs

```diff
@@ -317,10 +317,14 @@ impl HummockEventHandler {
                     }
 
                     HummockEvent::VersionUpdate(version_payload) => {
-                        self.local_version_manager
-                            .try_update_pinned_version(version_payload);
-
-                        // TODO update the ReadVersion
+                        if let Some(new_version) = self
+                            .local_version_manager
+                            .try_update_pinned_version(version_payload)
```

Comment:
IIUC, after the refactoring is fully done, we only need to update `read_version` on handling this event without touching other components (e.g. `local_version_manager` will be gone). Is this correct?

Reply:
yes, its quick workaround before we before we finish the sequential apply task. In short, we now only update commited_version and not staging_local_sst.

after the refactoring is fully done, event_handler will update the ReadVersion for all instance with Delta of Snapshot,  I think that's when the local_version_manager will be gone. (I'm not quite sure if we need to have a separate component to maintain a full PinnedVersion, since ReadVersion is per instance. Maybe it can be matained in global_hummock)